internal/indexmanagement/scripts.go

```diff
@@ -122,31 +123,79 @@ for i in range(0, len(indices), 25):
   print(','.join(indices[i:i+25]))
 `
 
+const indexManagement = `
+
+CONNECT_TIMEOUT=${CONNECT_TIMEOUT:-30}
+
+function getWriteIndex() {
+
+  local policy="$1"
+
+  # find out the current write index for ${POLICY_MAPPING}-write and check if there is the next generation of it
+  aliasResponse=$(curl -s $ES_SERVICE/_alias/${policy} \
+    --cacert /etc/indexmanagement/keys/admin-ca \
+    --connect-timeout ${CONNECT_TIMEOUT} \
+    -H"Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" \
+    -HContent-Type:application/json \
+    --retry 5 \
+    --retry-delay 5)
+
+  if [ -z "$aliasResponse" ]; then
+    echo "Received an empty response from elasticsearch -- server may not be ready"
+    exit 1
+  fi
+
+  jsonResponse="$(echo [$aliasResponse] | sed 's/}{/},{/g')"
+
+  if ! writeIndices="$(python /tmp/scripts/getWriteIndex.py "${policy}" "$jsonResponse")" ; then
```

Comment:
Can we spell out the `echo $writeIndices` as a proper exit message to understand what failed where?

Reply:
the python script should do this for us already